docs/partner_editable/additional_info.adoc

```diff
@@ -1,21 +1,21 @@
 == Test the deployment
 
-When you enroll a new AWS Control Tower–managed account, the deployment sets up the IAM role for {partner-product-short-name}. You can then see the new account show up in the New Relic UI.
+When you enroll a new AWS Control Tower–managed account, the deployment sets up the IAM role and Metrics Streaming Resources for {partner-product-short-name}. You can then see the new account show up in the New Relic UI.
 
 . https://one.newrelic.com[Log in^] to your New Relic account.
 . To view your AWS data, see https://docs.newrelic.com/docs/integrations/amazon-integrations/get-started/introduction-aws-integrations/#insights[Introduction to AWS integrations^].
 
 == Cleanup
 If you want to deploy this Quick Start for testing or demonstration purposes, and you don’t intend to use New Relic AWS integrations any longer, follow these steps to remove the AWS CloudFormation stack.
 
-WARNING: Deleting the {partner-product-short-name} Quick Start stack and StackSets also deletes the New Relic IAM roles deployed in your AWS accounts, which removes cross-account trust with New Relic. This Quick Start does not, however, remove account links to New Relic. To remove an account from New Relic, see the subsequent section *Uninstall the new Relic integration*. 
+WARNING: Deleting the {partner-product-short-name} Quick Start stack and StackSets also deletes the New Relic IAM roles deployed in your AWS accounts, which removes cross-account trust with New Relic. This Quick Start does not, however, remove account links to New Relic. To remove an account from New Relic, see the subsequent section *Uninstall the new Relic integration*.
 
 === Remove the AWS CloudFormation stack
 
 The time to complete this step depends on how many AWS accounts are included in your {partner-product-name} deployment. If deleting the AWS CloudFormation stack times out, it's safe to retry this step.
 
 . Sign in to the https://console.aws.amazon.com/console/home[AWS CloudFormation console^] in your Control Tower management account.
-. To remove all of the deployed AWS resources, delete the {partner-product-name} stack. This also deletes all stack set instances, which include any IAM roles deployed for enrolled AWS accounts. 
+. To remove all of the deployed AWS resources, delete the {partner-product-name} stack. This also deletes all stack set instances, which include any IAM roles and Metrics Streaming Resources deployed for enrolled AWS accounts. 
```

Comment:
Looking at the catalog, "Metrics Streaming Resources" isn't listed. Should this be [monitoring](https://docs.aws.amazon.com/controltower/latest/userguide/monitoring-overview.html)?

Reply:
No. This solution is not monitoring control tower. This solution is bootstrapping new accounts created by CT to ship metrics data using [CW metrics streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html)